README.md

```diff
@@ -85,3 +85,7 @@ Then, update your `composer.lock` file with:
     clu <git-url>
 
 The script provides sufficiently verbose output for debugging purposes.
+
+## GitLab support
+
+CLU supports GitLab in addition to GitHub. To use the project with a GitLab based repo, pass a `--gitlab` flag when running CLU.
```

Comment:
The `README` should mention the need for `lab` to be installed, instead of `hub`, for GitLab usage.